### PR TITLE
Prop 60 credentials clean up

### PR DESCRIPTION
**Pull request by dkosowski87**
## Changes
Moved credentials to envs.
Separated configuration from secrets.
Added secrets.yml to repo.
## Jira task
https://netguru.atlassian.net/browse/PROP-60

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gem 'rails', '~> 4'
 
+gem 'dotenv-rails', require: 'dotenv/rails-now', groups: %i(development test)
```
Comment (lubieniebieski) [inline Gemfile]: please add in an alphabetical order, there is a group `:development, :test` - please use it

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ def authenticate_user!
   end
 
   def check_domain!
-    return if AppConfig.single_domain
+    return if AppConfig.single_domain_mode.present?
```
Comment (lubieniebieski) [inline app/controllers/application_controller.rb]: 👏 I like being precise in regards to bool values!

File: config/environments/production.rb [hunk 3]
```diff
@@ -90,7 +90,7 @@
   }
 
   # ActionMailer Config
-  config.action_mailer.default_url_options = { :host => AppConfig.domain_name }
+  config.action_mailer.default_url_options = { :host => AppConfig.app_domain }
```
Comment (lubieniebieski) [inline config/environments/production.rb]: how about `host: AppConfig.app_domain` ?

File: config/environments/staging.rb [hunk 4]
```diff
@@ -93,7 +93,7 @@
   }
 
   # ActionMailer Config
-  config.action_mailer.default_url_options = { :host => AppConfig.domain_name }
+  config.action_mailer.default_url_options = { :host => AppConfig.app_domain }
```
Comment (lubieniebieski) [inline config/environments/staging.rb]: same as above

File: Gemfile (hunk 1 above)
Comment (dkosowski87) [inline Gemfile]: In order to use the `dotenv-rails` gem with `app_konfig` we need to require it before loading appkonfig (as AppConfig merges the config on initialization). This means that `dotenv-rails` should be listed before the  `app_konfig` gem in the Gemfile.

**Pull request merged**